prac_06/guitar.py

```diff
@@ -0,0 +1,26 @@
+"""Guitar class."""
+
+CURRENT_YEAR = 2023
+VINTAGE_MINIMUM = 50
```

Comment:
Could this constant name be slightly more specific by including the word 'years'?